troi/patches/daily_jams.py

```diff
@@ -42,13 +43,18 @@ def read(self, inputs = []):
         num_per_day = len(recordings) // 7
         days = [recordings[i:i + num_per_day] for i in range(0, len(recordings), num_per_day)]
 
-        return days[self.day - 1]
+        jam_date = datetime.utcnow() - timedelta(days=datetime.utcnow().isoweekday() % 7)
+        jam_date += timedelta(days=self.day)
+        jam_date = jam_date.strftime("%Y-%m-%d %a")
 
+        return [ Playlist(name="Daily Jams for %s, %s" % (self.user, jam_date), recordings=days[self.day - 1]) ]
 
-class DailyJamsPatch(troi.patch.Patch):
 
-    def __init__(self, debug=False):
-        super().__init__(debug)
+class DailyJamsPatch(troi.patch.Patch):
+    """
+        Taken a list of Recordings, break them into 7 roughly equal chunks and return
+        the chunk for the given day of the week.
```

Comment:
Does this return a `Playlist` with one playlist for the given day?  Is there a way that it makes sense for this to return the 7 playlists and allow us to just index into it given the day? (I've not looked too much at the implementation of this pipeline, so I'm not sure if it's possible or a good idea)

Reply:
Possibly. I think all of this is still in flux that I don't want to mess with it right now.